atomdb/periodic.py

```diff
@@ -1,210 +1,228 @@
+        "    Atomic number.\n"
+        "symbol : str\n"
+        "    Element symbol.\n"
+        "name : str\n"
+        "    Element name.\n"
+    )
+
+    # Autocomplete class docstring with data from the CSV files
+    for prop, name in prop2name.items():
+
+        # Add signature, description, sources, units, urls, and notes
+        short = f"{name} of the element."
+        if len(prop2col[prop]) == 1 and "" in prop2col[prop]:
+            # Only one default source
+            t = type(data[0][prop2col[prop][""]]).__name__
+            sig = f"{prop} : {t}"
+            long = ""
+        else:
+            # Multiple or non-default sources
+            t = type(data[0][next(iter(prop2col[prop].values()))]).__name__
+            sig = f"{prop} : Dict[{t}]"
+            long = "\n" "Notes\n" "-----\n" "This property is a dictionary with the following keys:"
+            # Add unit, url, note for each source
+            for src in prop2col[prop].keys():
+                long += f'\n    * "{src}"'
+                if prop2src[prop][src] != "":
+                    long += "\n        * Source\n" f"{indent_lines(prop2src[prop][src], 12)}"
+                if prop2unit[prop][src] != "":
+                    long += "\n        * Units\n" f"{indent_lines(prop2unit[prop][src], 12)}"
+                if prop2url[prop][src] != "":
+                    long += "\n        * URL\n" f"{indent_lines(prop2url[prop][src], 12)}"
+                if prop2note[prop][src] != "":
+                    long += "\n        * Notes\n" f"{indent_lines(prop2note[prop][src], 12)}"
+            long += "\n"
+
+        # Add property to class docstring
+        class_doc += f"{sig}\n" f"    {short}\n"
+
+        # Make property method for Element class with docstring
+        f = make_property(data, prop, prop2col)
+        f.__doc__ = f"{short}\n" "\n" "Returns\n" "-------\n" f"{sig}\n" f"{long}"
+
+        # Add property method to attributes
+        attrs[prop] = f
+
+    # Add class docstring to attributes
+    attrs["__doc__"] = class_doc
+
+    # Construct Element class
+    Element = type("Element", (object,), attrs)
+
+    # Return constructed class and functions
+    return Element, element_number, element_symbol, element_name
+
+
+def read_csv(file):
+    r"""Read a CSV file into a list of lists."""
+    lines = []
+    with open(file) as f:
+        for row in reader(f):
+            # ignore comments and empty lines
+            if row and not row[0].lstrip().startswith("#") and any(c not in ", \n" for c in row):
+                # Replace \n with new line in each element
+                lines.append([i.replace("\\n", "\n") for i in row])
+    return lines
+
+
+def get_data():
+    r"""Extract the contents of ``data/elements_data.csv``."""
+    data = read_csv(files("atomdb.data").joinpath("elements_data.csv"))
+    # Get property keys/source keys/units
+    props = data.pop(0)[3:]
+    srcs = data.pop(0)[3:]
+    units = data.pop(0)[3:]
```

Comment:
@msricher can you incorporate the conversion of the data to atomic units to this? 
Was this part in the current version in master:
https://github.com/theochem/AtomDB/blob/a09667edb02a53ce990e44777115d31f75e63f99/atomdb/periodic.py#L50-L55

Reply:
I will do this.

Reply:
@gabrielasd OK It's done.